nixos/modules/system/activation/switch-to-configuration.pl

```diff
@@ -412,6 +412,21 @@ sub filterUnits {
 # Make systemd reload its units.
 system("@systemd@/bin/systemctl", "daemon-reload") == 0 or $res = 3;
 
+# Reload user units
+opendir (DIR, "/run/user") or $res = 3;
```

Comment:
Shouldn't this use `loginctl` to get active sessions?

Reply:
I'll look into this. I thought of reading `/run/user` first as easiest approach to get active sessions, but if there's an easier way to implement this in perl with `loginctl` I'll change this!